wallet-new/src/Cardano/Wallet/API/Request/Filter.hs

```diff
@@ -35,15 +41,21 @@ import           Cardano.Wallet.API.V1.Types
 -- the inner closure of 'FilterOp'.
 data FilterOperations a where
     NoFilters  :: FilterOperations a
-    FilterOp   :: (IndexRelation a ix, FromHttpApiData ix, ToHttpApiData ix, Eq ix)
+    FilterOp   :: ( IndexRelation a ix
+                  , KnownSymbol (IndexToQueryParam a ix)
+                  , BuildableSafe ix
+                  , FromHttpApiData ix
+                  , ToHttpApiData ix
+                  , Eq ix
```

Comment:
Hopefully that's the last bunch of constraints we need to add, this is rapidly turning into a kitchen sink 😁 :trollface: